src/main/kotlin/de/evoila/osb/checker/util/MyTreePrinter.kt

```diff
@@ -9,129 +9,149 @@ import org.junit.platform.engine.reporting.ReportEntry
 import java.io.PrintWriter
 
 class MyTreePrinter(
```

Comment:
Changes are very hard to read, so take my review here with a grain of salt, but LGTM.